course-mw/enrolment-actor/src/main/scala/org/sunbird/group/GroupAggregatesActor.scala

```diff
@@ -0,0 +1,144 @@
+package org.sunbird.group
+
+import org.apache.commons.collections.CollectionUtils
+import org.apache.commons.collections.MapUtils
+import org.apache.commons.lang3.StringUtils
+import org.sunbird.common.exception.ProjectCommonException
+import org.sunbird.common.models.response.Response
+import org.sunbird.common.models.util.{ProjectLogger}
+import org.sunbird.common.request.Request
+import org.sunbird.common.responsecode.ResponseCode
+import org.sunbird.keys.SunbirdKey
+import org.sunbird.learner.actors.group.dao.impl.GroupDaoImpl
+import org.sunbird.actor.base.BaseActor
+import org.sunbird.cache.CacheFactory
+import org.sunbird.cache.interfaces.Cache
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+class GroupAggregatesActor extends BaseActor {
+
```

Comment:
Inject the dependencies. Override onClose method and close the dependencies safely. For ex: A cassandra connection

Reply:
And a redis connection